test/universalfallback.jl

```diff
@@ -64,8 +65,12 @@ end
 
 struct UnknownOptimizerAttribute <: MOI.AbstractOptimizerAttribute end
 
+# A few constraint types are supported to test both the fallback and the
```

Comment:
Nit: "A few" -> "Few"

Reply:
To my Kiwi English, it seems weird to say "Few" here. To me, "A few" reads as "we've only implemented a couple of constraint types to test the functionality" whereas "Few" has a negative connotation and means "Not enough constraint types are supported".

[SO suggests](https://english.stackexchange.com/questions/1866/a-few-vs-few) the following example:
 - I have few friends.
 - I have a few friends.

Reply:
I meant "A few" in opposition of none otherwise only the fallback is tested